index.html

```diff
@@ -1,46 +1,153 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0"> <!-- displays site properly based on user's device -->
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- displays site properly based on user's device -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600;700&family=Poppins:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="./images/favicon-32x32.png"
+    />
+    <link rel="stylesheet" href="styles/style.css" />
+    <title>Frontend Mentor | Huddle landing page with curved sections</title>
+  </head>
+  <body>
+    <main class="container">
+      <header>
+        <img class="logo" src="./images/logo.svg" alt="" />
+        <div class="try-it-free-container">
+          <a class="try-it-free" href="#">Try It Free</a>
+        </div>
+      </header>
+    </main>
+    <div class="section-1-hero">
+      <h1 class="title">Build The Community Your Fans Will Love</h1>
+      <p class="title-description title">
```

Comment:
Two classes here, good one.
A bit of an overkill here and not needed, but, the title content could be put in a div with the class title. Then, you could do:
`.title > * {
}`

Again, not needed here but a good way to get used to logically separate pieces of html to make it easier for you in the future